package.json

```diff
@@ -57,6 +57,8 @@
         "@babel/preset-typescript": "^7.3.3",
         "@babel/register": "^7.5.5",
         "@colony/colony-js-adapter": "^1.14.0-beta.2",
+        "@colony/eslint-config-colony": "^8.0.0",
+        "@kironeducation/flow-junit-transformer": "^0.3.0",
```

Comment:
How is this package used?

Reply:
I have no idea why this is in there. I’ll fix

Reply:
Done